internal/storage/rocksdb/store.go

```diff
@@ -161,6 +196,20 @@ func (rdb *rocksDB) Close() error {
 	return nil
 }
 
+func (rdb *rocksDB) PutTTL(key []byte, value []byte, expireTS int64) error {
+	defer rdb.opts.statsCli.Timing("rocksdb.put.latency.ms", time.Now())
+	if expireTS > 0 {
+		b := make([]byte, tsLength)
+		binary.LittleEndian.PutUint64(b, uint64(expireTS))
+		value = append(value, b...)
```

Comment:
Default precision of Unix epoch is till milliseconds. But if we just constrain/assume for users to just use minutes or seconds level precision, we can reduce the representation to < 8 bytes. Perhaps even just 4 bytes (int). Please check.

Reply:
This is already seconds https://golang.org/pkg/time/#Time.Unix. IMO we should allow second accuracy to allow users to have the choice to set 90sec ttl and not forced to use 120sec, which would happen if we move to minute level.